### PR TITLE
docs: fix doc string in `value_object` package

### PR DESCRIPTION
**Pull request by rokamu623**
## Description
Added and corrected missing information about doc string in `value_object` package.

## Related links
Jira: https://tier4.atlassian.net/browse/RT2-258

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been properly tested.
- [ ] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] (Optional) The PR has been properly tested with CARET_report verification.
- [ ] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

File: src/caret_analyze/value_objects/publisher.py [hunk 1]
```diff
@@ -38,28 +55,82 @@ def __init__(
 
     @property
     def node_id(self) -> str:
+        """
+        Construct an instance.
```
Comment (taro-yu) [inline src/caret_analyze/value_objects/publisher.py:59]: Is there a "Get node id" mistake here?

File: src/caret_analyze/value_objects/callback.py [hunk 2]
```diff
@@ -356,6 +460,32 @@ def __init__(
...
+        symbol : str
+            Symbol name of the service callback.
+        service_name : str
+            Service name which the service callback service.
+        publish_topic_names : tuple[str, ...] | None
+            Topic name which the service callback publishes.
+        construction_order : int
+            Order of instance creation within the identical node.
+        callback_name : str | None
+            Service callback name, by default None.
```
Comment (taro-yu) [inline src/caret_analyze/value_objects/callback.py:485]: It is better that "Service callback name, by default None." is "Service callback name (: default is None)." 
However, there are many same sentence (, by default None), so if changes are made in this way, everything needs to be changed.

**Review by emori-ctc — APPROVED** [review]

File: src/caret_analyze/value_objects/publisher.py (hunk 1 above)
Comment (rokamu623) [reply src/caret_analyze/value_objects/publisher.py]: Fix it in [8a29ff2](https://github.com/tier4/caret_analyze/pull/454/commits/8a29ff281113693de018ca4ac42f026ba7bc0e4d).

File: src/caret_analyze/value_objects/callback.py (hunk 2 above)
Comment (rokamu623) [reply src/caret_analyze/value_objects/callback.py:485]: This grammar has been used for previous, so there is no need to change it this time, I think.

Comment (taro-yu) [reply src/caret_analyze/value_objects/callback.py:485]: I see. Then I don't think a fix is needed.

**Review by taro-yu — APPROVED** [review]
LGTM!

**Review by emori-ctc — APPROVED** [review]
LGTM!

**Pull request merged**